test/wrapper_array.jl

```diff
@@ -72,6 +72,37 @@ end
         @test size(nda[CartesianIndex(1, 1), newaxis]) == (1, )
         @test nda[CartesianIndex(1, 1), newaxis][1] == nda[1, 1]
     end
+
+    # https://github.com/invenia/NamedDims.jl/issues/70
+    @testset "BitArray" begin
+        nda = NamedDimsArray(rand(1:9, 3,3,3), (:x, :y, :z))
+        @test dimnames(nda[rand(3) .> 0.3, 1, :]) == (:x, :z)
+
+        # flattening of ndims>1 should drop names
+        @test dimnames(nda[rand(3,3,3) .> 0.3]) == (:_,)
+        @test dimnames(nda[nda .> 3]) == (:_,)
+
+        # but vectors are unambiguous, so keep theirs
```

Comment:
Can we break each of these, here and below into nested testsets?

```suggestion
        @testset "vectors are unambiguous, so keep their names" begin
```